### ISSUE TITLE
insert into values can use substring?

### ISSUE THREAD

**Issue by zhangwangren** [apache/shardingsphere#14465]
## Question

**For English only**, other languages will not accept.
Excuse me! In the master branch, can i use the function SUBSTRING() in the INSERT INTO VALUES? Like this
INSERT INTO tb_user(user_no, user_name) VALUES (#{userNo}, SUBSTRING(#{userName}, 1, 2))，(#{userNo}, SUBSTRING(#{userName}, 1, 2));

**Comment by terrymanu** [comment]
Can you try it?

**Comment by zhangwangren** [comment]
> Can you try it?

Yes! it throw a Exception like this "Cause: java.sql.SQLException: No value specified for parameter";

**Comment by terrymanu** [comment]
No feedback, closed
